Update SQL module release notes (#1125)
ballerina-platform/ballerina-release: release-notes/swan-lake-beta3-release-note.md

int count = sqlClient->queryRow(`SELECT COUNT(*) FROM ExTable`);
```

> - Introduced `queryConcat()` and `arrayFlattenQuery()` util functions to dynamically create a complex query.

Reviewer: ```suggestion
- Introduced `queryConcat()` and `arrayFlattenQuery()` util functions to create a complex query dynamically.
```

>   - The `queryConcat()` creates a parameterized query by concatenating a set of parameterized queries.

Reviewer: ```suggestion
  - The `queryConcat()` function creates a parameterized query by concatenating a set of parameterized queries.
```

    ```ballerina
    sql:ParameterizedQuery query = `SELECT * FROM students`;
    sql:ParameterizedQuery query1 = ` WHERE id < ${id} AND age > ${age}`;
    sql:ParameterizedQuery sqlQuery = sql:queryConcat(query, query1);
    ```

>   - The `arrayFlattenQuery()` is introduced to make the array flatten easier. It makes the inclusion of 

Reviewer: ```suggestion
  - The `arrayFlattenQuery()` is introduced to make the array flattening easier. It makes the inclusion of 
```

    varying array elements into the query easier by flattening the array to return a parameterized query.
    ```ballerina
    int[] ids = [1, 2];
    sql:ParameterizedQuery sqlQuery = sql:queryConcat(`SELECT * FROM DataTable WHERE id IN (`, sql:arrayFlattenQuery(ids), `)`);
    ```

##### WebSocket Package

> - Added OAuth2 JWT bearer grant type support for client

Reviewer: ```suggestion
- Added OAuth2 JWT bearer grant type support for the client
```

int count = sqlClient->queryRow(`SELECT COUNT(*) FROM ExTable`);
```

> - Introduced `queryConcat()` and `arrayFlattenQuery()` util functions to create a dynamic/constant complex query.

Reviewer 2: ```suggestion
- Introduced `queryConcat()` and `arrayFlattenQuery()` util functions to dynamically create a complex query.
```

  - The `queryConcat()` creates a parameterized query by concatenating a set of parameterized queries.
    ```ballerina

>     int id = 10;
>     int age = 12;

Reviewer 2: ```suggestion
```

Reviewer 2: As we need to show how two queries append, we can skip these variable
